dishes/colombian/otras_preparaciones/envueltos.md

```
@@ -1,108 +1,33 @@
-- "El secreto es usar hojas frescas y masa bien hidratada."
-- "Puedes rellenar con bocadillo o queso para versiones especiales."
-- "Perfectos para acompañar bebidas calientes en el desayuno."
+  - "https://www.mycolombianrecipes.com/es/envueltos-de-mazorca/"
+  - "https://www.recetasgratis.net/receta-de-envueltos-de-mazorca-23754.html"
+license: "MIT"
+yaml_validation: "https://yaml-online-parser.appspot.com/"
+---
+
+## Ingredientes
+
+*   10 mazorcas tiernas y grandes
```

markdownlint-cli2 (0.17.2)

Spaces after list markers
Expected: 1; Actual: 3

(MD030, list-marker-space)